test/demo/accounts_test.exs

```diff
@@ -0,0 +1,461 @@
+defmodule Demo.AccountsTest do
+  use Demo.DataCase
+
+  import Demo.AccountsFixtures
+  alias Demo.Accounts
+  alias Demo.Accounts.{User, UserToken}
+
+  describe "get_user_by_email/1" do
+    test "does not return the user if the email does not exist" do
+      refute Accounts.get_user_by_email("unknown@example.com")
+    end
+
+    test "returns the user if the email exists" do
+      %{id: id} = user = user_fixture()
+      assert %User{id: ^id} = Accounts.get_user_by_email(user.email)
+    end
+  end
+
+  describe "get_user_by_email_nad_password/1" do
```

Comment:
```suggestion
  describe "get_user_by_email_and_password/1" do
```

Reply:
Thanks! I am done with the contexts and the auth module. Tomorrow I should be done with controllers. I am not planning to unit test views and templates, so those should be safe for converting into generators. :)

Reply:
This is looking awesome! :heart: 

I'm copying code across into the generators and plan to run a diff between your branch and the code the generators create to see if I've missed anything.